pkg/vetter/serviceportprefix/README-missing-service-port-prefix.md

```diff
@@ -32,4 +32,5 @@ In version 0.2.12, these protocols are supported: `grpc`, `https`, `http2`,
 
 ## See Also
 
-- [Service Requirements](https://istio.io/docs/setup/kubernetes/spec-requirements/)
+- [Service
```

Comment:
Change ` In version 0.2.12, these protocols are supported: `grpc`, `https`, `http2`,
 `http`, `tcp`, `udp`, `mongo`, `redis`.` to  `In version 0.8.0, these protocols are supported: `grpc`, `https`, `http2`, `http`, `tcp`, `udp`, `mongo`, `redis`.`